src/foundry/foundry.js/collections/documentCollections/worldCollections/actors.d.ts

```diff
@@ -0,0 +1,72 @@
+import { ConfiguredDocumentClass } from '../../../../../types/helperTypes';
+
+declare global {
+  /**
+   * The singleton collection of Actor documents which exist within the active World.
+   * This Collection is accessible within the Game object as game.actors.
+   *
+   * @see {@link Actor} The Actor entity
+   * @see {@link ActorDirectory} The ActorDirectory sidebar directory
+   *
+   * @example <caption>Retrieve an existing Actor by its id</caption>
+   * ```typescript
+   * let actor = game.actors.get(actorId);
+   * ```
+   */
+  class Actors extends WorldCollection<typeof foundry.documents.BaseActor, 'Actors'> {
+    constructor(data?: InstanceType<ConfiguredDocumentClass<typeof foundry.documents.BaseActor>>['data']['_source'][]);
```

Comment:
Default value?

Reply:
It's the same as for `WorldCollection`, so I think it's not necessary to document it again